scripts/lib/execution-utils.js

```diff
@@ -27,6 +27,7 @@ function runMain(mainFunction) {
     .catch((error) => {
       printError('\nScript exited with error:')
       printError(error)
+      printError(error.cause)
```

Comment:
**💬 suggestion:** ‏I think we can do a bit better, because not all errors have a cause (so your code will print "undefined" most of the time), and error causes can also have a cause (it's recursive). Let's create a new function, `printErrorWithCause(error)` that prints error causes. Usage example:

```ts
const error = new Error('Foo', { cause: new Error('Bar', { cause: new Error('Baz') }) })
printErrorWithCause(error)
// Prints:
// Error: Foo
// Caused by: Error: Bar
// Caused by: Error: Baz
```